core/spec/models/spree/line_item_spec.rb

```diff
@@ -188,4 +188,12 @@
       expect(line_item.price).to eq 21.98
     end
   end
+
+  describe "precision of pre_tax_amount" do
+    let!(:line_item) { create :line_item, pre_tax_amount: 4.2051 }
+
+    it "keeps four digits of precision even when reloading" do
```

Comment:
This commit is part of a larger refactoring of the taxation system. Are you planning to also port that to Solidus?